chromadb/api/segment.py

```diff
@@ -789,7 +791,7 @@ def reset(self) -> bool:
     def get_settings(self) -> Settings:
         return self._settings
 
-    @property
+    @cached_property
```

Comment:
is there a reason for this? it adds a small amount of cognitive overhead ("what's a cached property?") for no benefit that i can see?